### DISCUSSION TITLE
Issues with VST and Ctrlr Panel

### DISCUSSION THREAD

**Discussion by pchclx** [inteyes/ReMS2000#4] [Q&A]
Hi, this is an amazing tool and I'd love to get into using it.  I'm having a couple issues.

Both versions of the VST Plugin are not detected by my DAW / Host (in this case, Ableton 11.1 x64.  Other VST Plugins seem to be detected fine.

I tried instead using Ctrlr VST plugin and opening the ReMS2000 panel within, but get the below errors.

![image](https://user-images.githubusercontent.com/4944194/215301499-8625b413-f82e-4ea6-8a58-ec3fec4b0ac2.png)

![image](https://user-images.githubusercontent.com/4944194/215301513-1fbe3c85-29c1-4a9e-947a-4421e421b015.png)

Ctrlr is v 6.0.4 Jan 16 2019.  
The ReMS2000 panel is v 1.3.4.  I see the ReMS2000 panel says "Created with Ctrlr 5.3.201", I wonder if it will work if I install that older version of Ctrlr?

The only reason I know I'm not completely crazy is that  ReMS2000 standalone .EXE works perfectly.  I just can't seem to use it from within a DAW.  Any help appreciated, and thanks for making this awesome piece of software.

**Comment by inteyes** [comment]
Hi, @pchclx!

As far as I know, version 5.3.201 is the last stable and recommended by the Ctrlr community version of Ctrlr. So yeah, there is a good chance that everything will work with that particular version (5.3.201) as expected.

I don't have much experience with Ableton, so I can only guess here. The exported instances for this panel are 32 / 64 bit Windows VST2 libraries. Please check if "VST2 Plug-in Custom Folder" settings item is enabled and point to the right destination where ReMS2000_x64.dll is present. 

Tested with a Trial Ableton 11.2.7 x64 and it worked fine for me, the panel appeared in the list of plugins after scanning.


![untitled](https://user-images.githubusercontent.com/31481722/215333546-f22fbca0-de96-4f45-9f2d-eb7e915c8421.png)

**Reply by bespokecomp** [reply]
Can I check something here?
Does Ctrlr need to be installed first, for this to all work?
This is what I do t quite understand. 

Also, exactly what should the hardware configuration be?

Interface MIDI Out --> MS2000 MIDI In
            &
MS2000 MIDI Out    --> Interface MIDI In

??

Many thanks

**Reply by inteyes** [reply]
Hey there. 

There is no need to install Ctrlr unless you want to modify this panel in any way or if you are using a non-Windows environment (but which is supported by Ctrlr itself). 

If you are using Windows, it will be enough to drop the appropriate dll into the VST Plugins folder and rescan the plugins inside your DAW. In this case, ReMS2000_* should appear and can now be used like any other VST plugin.

> Also, exactly what should the hardware configuration be?
> 
> Interface MIDI Out --> MS2000 MIDI In & MS2000 MIDI Out --> Interface MIDI In

Yes, that's correct. This panel requires a two-way connection to operate. It is also necessary to set MIDI Input / Output devices and channels in the panel settings.
